src/utils/getUserRewards.ts

```diff
@@ -54,16 +62,16 @@ export const getUserRewards = async (
     claimed = await rewardsDisributor.claimed(address);
     claimData = claimable.sub(claimed).gt(0)
```

Comment:
Can this really happen without being a problem?

Reply:
wdym?

Reply:
claimable is the amount that user can claim since the beginning of the first distribution. claimed is the amount that user has already claimed